Automattic/jetpack
# Jetpack Connection banner: update visual

Fixes #31439

## Proposed changes:

Update our connection banner visuals.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* On a brand new site, go to the main wp-admin dashboard, before the site is connected.
* Check that the banner looks good at different screen resolutions.
